app/message-pin/client/views/pinnedMessages.js

```diff
@@ -37,9 +37,6 @@ Template.pinnedMessages.onCreated(function() {
 			_hidden: { $ne: true },
 			pinned: true,
 			rid: this.data.rid,
-			_updatedAt: {
-				$gt: new Date(),
```

Comment:
Instead of removing the updated at, you can change the `$gt` with a `$gte`, this way we don't disrupt the intended functionality while fixing the issue. Thanks